pom.xml

```diff
@@ -1117,6 +1123,11 @@
                     </property>
                 </activation>
             <repositories>
+                <repository>
+                    <id>mvnrepository</id>
+                    <name>mvnrepository</name>
+                    <url>https://mvnrepository.com/artifact/</url>
+                </repository>
```

Comment:
Which dependency needs this?

Reply:
It looks like google-cloud-bigquery exists in central...

Reply:
OK, that may not be required. I had to add it to build locally. Maybe my maven settings are different.